bootstrap/terraform/aws-bootstrap/existing.tf

```diff
@@ -0,0 +1,9 @@
+data "aws_eks_cluster" "cluster" {
+    count = var.create_cluster ? 0 : 1
+    name = var.cluster_name
```

Comment:
Our cluster name variable is limited to 15 characters through the CLI, but they can be longer in AWS. When deploying a cluster through the Cluster-API it will add some prefixes to cluster names so they will be longer than our CLI supports. We should allow users to select from existing clusters if they choose to use BYOK in our CLI during the `plural init` stage.